test/hotspot/jtreg/compiler/loopopts/superword/ProdRed_Double.java

```diff
@@ -101,6 +81,10 @@ public static void prodReductionInit(double[] a, double[] b) {
         }
     }
 
+    //8300865 : No reduction nodes emitted (x86_64)
```

Comment:
We try to avoid references to issue numbers in the code since they do not tend to age well (an exception is of course the top-level `@bug` annotation). I think it is valuable to have a comment explaining why you do not currently expect any vectorization in this case, but could you rewrite it without the explicit issue reference?